<launch>
    <arg name="cmd_vel_topic" default="/cmd_vel"/>
    <node pkg="bugbase_node" exec="odrive_node" name="odrive_node" namespace="bugbase_bringup">
        <param from="$(find-pkg-share bugbase_node)/config/rc_control.yaml"/>
        <remap from="cmd_vel" to="$(var cmd_vel_topic)"/>
    </node>
</launch>